pkg/config/context.go

```diff
@@ -13,6 +13,7 @@ const (
 	ContextOptionDotfilesScript             = "DOTFILES_SCRIPT"
 	ContextOptionSSHAgentForwarding         = "SSH_AGENT_FORWARDING"
 	ContextOptionSSHConfigPath              = "SSH_CONFIG_PATH"
+	ContextOptionExecuteTimeout             = "EXECUTE_TIMEOUT"
```

Comment:
Could you rename this to something that includes `agent` and `inject` to indicate which part of DevPod it's affecting? Otherwise it's hard to understand where exactly this context option comes into play 🤔

Reply:
done :)